src/main/resources/application.properties

```diff
@@ -35,22 +35,30 @@ grpc.server.keep-alive-time=30s
 grpc.server.keep-alive-timeout=15s
 grpc.server.permit-keep-alive-without-calls=true
 grpc.server.security.enabled=false
+grpc.server.security.certificate-chain=file:/grpc/server.crt
+grpc.server.security.private-key=file:/grpc/server.key
```

Comment:
is this needed if grpc.sever.security.enabled=false? I've seen it in other microservices as well.

Reply:
Yes, because these values must be in shared volume "/grpc" as defined in gradle jib. If we have to define location/file in every profile there is a greater chance of error. For now, I would leave like this beacuse it provides cleaner configuration for staging and demo.